code/modules/mob/living/carbon/brain/MMI.dm

```diff
@@ -48,9 +55,11 @@
 		newbrain.brainmob = null
 		brainmob.loc = src
 		brainmob.container = src
-		brainmob.stat = 0
-		dead_mob_list -= brainmob //Update dem lists
-		living_mob_list += brainmob
+		if(brainmob.health > config.health_threshold_dead)
+			brainmob.stat = CONSCIOUS
+			brainmob.reset_perspective()
+			dead_mob_list -= brainmob //Update dem lists
+			living_mob_list += brainmob

```

Comment:
Since the brain mob's stat is always DEAD (because brain code is shit), we need to check for its health directly so people can no longer beat an mmi to death, then eject the brain and put it back in the mmi to magically ressuscitate it.